core/PRP/Avatar/plAGMasterMod.cpp

```diff
@@ -74,6 +74,15 @@ void plAGMasterMod::write(hsStream* S, plResManager* mgr) {
     }
 }
 
+bool plAGMasterMod::orderAfter(const hsKeyedObject* other) const {
+    // This should be ordered after the plAGModifier of the same name
+    if (other->getKey()->getType() == kAGModifier
+            && other->getKey()->getName() == getKey()->getName()) {
```

Comment:
Considering that this is run over the entire list of modifiers, the name comparison could likely be dropped.

Reply:
The name comparison was intended to only create a strong dependency between associated keys.  For example, I assume it's perfectly fine for `[plAGModifier("cAnim01"), plAGMasterMod("cAnim01"), plAGModifier("cAnim01"), plAGMasterModifier("cAnim02")]` to remain interleaved, rather than sorting all AGModifiers before all AGMasterMods.

Reply:
It is, I'm just hoping there aren't any cases where the AGMaster and AGMod don't have the same key name. That's extremely unlikely IMO

Reply:
My concern here was more about not reordering things that don't need to be reordered (i.e. clean round-tripping of existing, working PRP files).  Is the association stored somewhere else that it can be referenced for key sorting?

Reply:
I highly doubt it, unfortunately